packages/node/src/lib/object-to-array.ts

```diff
@@ -1,14 +1,24 @@
 import type { URLSearchParams } from 'url';
 
+type ParamArray = {
+  name: string;
+  value: unknown;
+}[];
+
+export function objectToArray(
```

Comment:
Why export duplicate functions here?

Reply:
it's called a [function overload](https://www.typescriptlang.org/docs/handbook/2/functions.html#function-overloads)! since this function accepts an optional `opts` object, [the first overload](https://github.com/readmeio/metrics-sdks/blob/82f06dcff87312c8495b0a1d15d55c8ce4609b97/packages/node/src/lib/object-to-array.ts#L8-L11) allows us to specify a more specific return type :chefs-kiss:

_(this is literally my first time using them and I love 'em)_